src/components/navbar/Navbar.jsx

```diff
@@ -21,13 +22,14 @@ import {
 import { FaMoon, FaSun, FaSearch } from "react-icons/fa";
 
 const Navbar = (props) => {
+  console.log("Home", props);
   const { isOpen, onOpen, onClose } = useDisclosure();
 
   const handleToggle = () => (isOpen ? onClose() : onOpen());
   const { toggleColorMode } = useColorMode();
   const text = useColorModeValue("dark", "light");
   const SwitchIcon = useColorModeValue(FaMoon, FaSun);
-
+  const pathname = window.location.pathname;
```

Comment:
you should also be able to use a hook from `react-router-dom` to get the path instead of checking it on the window.

Example form stackoverflow:

`import { useLocation } from 'react-router-dom'

function HeaderView() {
  const location = useLocation();
  console.log(location.pathname);
  return <span>Path : {location.pathname}</span>
}
`

Reply:
fixed. thanks for the input